app/assets/fonts/SourceCodePro-Regular.ttf.license

```diff
@@ -0,0 +1,3 @@
+SPDX-FileCopyrightText: 2023 Adobe Systems Incorporated
+
+SPDX-License-Identifier: OFL-1.1
```

Comment:
Is it correct that I wrote this information here? If so, we need to do the same for the embedded Inter font.